pkg/broker/registry.go

```diff
@@ -77,7 +86,9 @@ func MapServiceIdToName() map[string]string {
 	out := map[string]string{}
 
 	for _, svc := range brokerRegistry {
-		out[svc.CatalogEntry().ID] = svc.Name
```

Comment:
Does `svc.CatalogEntry()` not return an error here because this is an older version of the library?

Reply:
This is me refactoring CatalogEntry so it re-parses the catalog entry on every call to be more idiomatic viper-y. It gets rid of all that pre-processing work done when you register a service at some extra CPU expense, but also readies us for no downtime config refreshes with only SIGHUP.